akka-projection-testkit/src/test/scala/akka/projection/testkit/ProjectionTestKitSpec.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.testkit
+
+import akka.{ Done, NotUsed }
+import akka.actor.ClassicActorSystemProvider
+import akka.actor.testkit.typed.scaladsl.{ ScalaTestWithActorTestKit, TestProbe }
+import akka.projection.Projection
+import akka.stream.{ KillSwitch, KillSwitches }
+import akka.stream.scaladsl.Source
+import akka.stream.scaladsl.{ Keep, Sink }
+import akka.stream.testkit.TestSubscriber
+import akka.stream.testkit.scaladsl.TestSink
+import org.scalatest.wordspec.AnyWordSpecLike
+import akka.actor.typed.scaladsl.adapter._
+
+import scala.concurrent.{ Future, Promise }
+
+class ProjectionTestKitSpec extends ScalaTestWithActorTestKit with AnyWordSpecLike {
+
+  val projectionTestKit: ProjectionTestKit = ProjectionTestKit(testKit)
+
+  "ProjectionTestKit" must {
+
+    "run a projection" in {
+      val probe = testKit.createTestProbe[Int]("projection-probe")
+      val prj = TestProjection(Source(1 to 5), probe)
```

Comment:
Here I'm just passing a general probe to inspect the projection. 

There are a few other ideas that I would like to experiment on another PR. For instance, have the possibility to run the projection with a `TestSink` so users can control the flow. In such a case, the user won't use an assert function but will control the elements and assert results after letting an element flow through the projection.